util/patricia_trie/src/node.rs

```diff
@@ -63,7 +63,7 @@ impl<'a> Node<'a> {
 			// an empty branch index.
 			Prototype::Data(0) => Ok(Node::Empty),
 			// something went wrong.
-			_ => Err(DecoderError::Custom("Rlp is not valid."))
+			_ => bail!("Rlp is not valid.")
```

Comment:
ah, sorry. I thought that this Error is already an error from error_chain. Wouldn't ask for this change if I knew it was not.